.travis/prepare_ezplatform.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+EZPLATFORM_BRANCH=`php -r 'echo json_decode(file_get_contents("./composer.json"))->extra->_ezplatform_branch_for_behat_tests;'`
+EZPLATFORM_BRANCH="${EZPLATFORM_BRANCH:-master}"
+PACKAGE_BUILD_DIR=$PWD
+EZPLATFORM_BUILD_DIR=${HOME}/build/ezplatform
+
+echo "> Cloning ezsystems/ezplatform:${EZPLATFORM_BRANCH}"
+git clone --depth 1 --single-branch --branch "${EZPLATFORM_BRANCH}" https://github.com/ezsystems/ezplatform.git ${EZPLATFORM_BUILD_DIR}
```

Comment:
Just a suggestion: could the repository name be a parameter (with default value "ezplatform") so we could run tests in parallel for ezplatform and ezplatform-ee?

Reply:
Won't work for -ee because we'd need to expose auth data (can be encrypted, but won't work for PRs from remote forks).

However nice feature, you could create separate job for demo meta repository if you want, implemented via 3068fd9.